src/transformers/tokenization_utils_base.py

```diff
@@ -3170,13 +3163,11 @@ def get_special_tokens_mask(
     @staticmethod
     def clean_up_tokenization(out_string: str) -> str:
         """
-        Clean up a list of simple English tokenization artifacts like spaces before punctuations and abbreviated forms.
+        clean up a list of simple english tokenization artifacts like spaces before punctuations and abbreviated forms.
 
-        Args:
-            out_string (:obj:`str`): The text to clean up.
+        args: out_string (:obj:`str`): the text to clean up.
 
-        Returns:
-            :obj:`str`: The cleaned-up string.
+        returns: :obj:`str`: the cleaned-up string.
```

Comment:
The docstring was in the correct style before the changes

Reply:
But I simply ran the documentation fixer :(